### PR TITLE
Update ugdg

### PR DESCRIPTION
**Pull request by bentanjunrong**
Let's:
- Remove more AB3 traces in docs
- Finally update the index page
- Standardise formatting for images in DG

Fixes:
- #187 
- #184 
- #185 (incomplete)

### CODE REVIEW

**Comment by bentanjunrong** [comment]
Not sure about how to do #185: Should the exam/lesson diagram be part of Model? I think it looks out of place if its on its own, but at the same time adding to Model might be too cluttered. Advice would be appreciated

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Refer to the guide [_Setting up and getting started_](SettingUp.md).
 
 ### Architecture
 
-<img src="images/ArchitectureDiagram.png" width="280" />
+![ArchitectureDiagram.png](images/ArchitectureDiagram.png)
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:32]: any reason for these changes? you want to make the images bigger for easier readability?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ The `UI` component,
 - keeps a reference to the current screen state as a `GuiState` in order to restrict some commands to certain screens (more on this under `Restricting Commands based on GUI state` in the `Implementation` section below).
 - changes the screen based on the return value of CommandResult.
 
-#### Design considerations:
+#### Design Considerations:
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:90]: remove the colon here

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ The sequence diagram of how this works for an `add lesson` command is found belo
 
 When storing the `ModBook`, the respective `JsonAdaptedObject` classes will create adaptations of objects in the `Model`, allowing `JsonUtil` to store the objects and save them to the `filePath` specified in `UserPrefs`.
 
-#### Design considerations:
+#### Design Considerations:
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:165]: remove colon

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -210,7 +210,7 @@ The sequence diagram of how this works for a `delete` command is found below. Th
 When parsing, the respective `Parser` will check the current `GuiState` with the allowed `GuiState`s. If the `GuiState` is valid, it will proceed with parsing the command.
 Otherwise, it will throw a `GuiStateException`.
 
-#### Design considerations:
+#### Design Considerations:
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:213]: remove colon

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -514,7 +514,7 @@ testers are expected to do more *exploratory* testing.
 
 </div>
 
-### Launch and shutdown
+### Launch and Shutdown
 
 1. Initial launch
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:519]: the numbering for this section needs to be updated as well

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -531,24 +531,24 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a module
+### Deleting a Module
 
 1. Deleting a module while all module are being shown
 
    1. Prerequisites: List all module using the `list` command. Multiple module in the list.
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:538]: change to
```suggestion
   1. Prerequisites: List all module using the `list mod` command. You must have at least one module in ModBook.
```

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -531,24 +531,24 @@ testers are expected to do more *exploratory* testing.
...
 
-   1. Test case: `delete 1`<br>
+   1. Test case: `delete mod 1`<br>
       Expected: First module is deleted from the list. Details of the deleted module shown in the status message. Timestamp in the status bar is updated.
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:541]: timestamp?

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -531,24 +531,24 @@ testers are expected to do more *exploratory* testing.
...
 
-   1. Test case: `delete 0`<br>
+   1. Test case: `delete mod 0`<br>
       Expected: No module is deleted. Error details shown in the status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   1. Other incorrect delete commands to try: `delete`, `delete 1`, `delete mod x`, `...` (where x is larger than the list size)<br>
```
Comment (rohit0718) [inline docs/DeveloperGuide.md:546]: i think can remove the `...`

File: docs/index.md [hunk 9]
```diff
@@ -1,17 +1,18 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: NUS ModBook
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```
Comment (rohit0718) [inline docs/index.md:6]: this needs to be updated as well

File: docs/index.md [hunk 10]
```diff
@@ -1,17 +1,18 @@
...
+NUS ModBook is a **desktop app for NUS students to manage modules, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ModBook can help you organise modules faster than traditional GUI apps.
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using NUS ModBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
+* If you are interested about developing NUS ModBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you're curious about the team behind NUS ModBook, check out the [**About Us**](AboutUs.html) page! 😃
 
 
 **Acknowledgements**
```
Comment (rohit0718) [inline docs/index.md:18]: perhaps we shd acknowledge ab3 here?

**Review by rohit0718 — CHANGES_REQUESTED** [review]
thank you for the help sir ❤️ . just some minor changes

**Comment by rohit0718** [comment]
> Not sure about how to do #185: Should the exam/lesson diagram be part of Model? I think it looks out of place if its on its own, but at the same time adding to Model might be too cluttered. Advice would be appreciated

Perhaps can put in Model section itself. First we show the general diagram (with the Exam and Lesson blocks). Then you can write something like: "For a more detailed diagram of Lesson and Exam, refer to the diagrams below:" just after the generic diagram? Then you can put both of the diagrams there as well.

File: docs/AboutUs.md [hunk 11]
```diff
@@ -5,47 +5,45 @@ title: About Us
...
 
 <img src="images/rohit0718.png" width="200px">
 
-[[github](https://github.com/rohit0718)]
+[[GitHub](https://github.com/rohit0718)]
 
 * Role: Documentation
 
-### TAN JUN RONG, BENJAMIN
+### Tan Jun Rong, Benjamin
```
Comment (luminousleek) [inline docs/AboutUs.md:34]: uhh want to standardised and just put Benjamin Tan?

**Review by luminousleek — CHANGES_REQUESTED** [review]
minor changes, rohit got most of them. thanks sir!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (bentanjunrong) [reply docs/DeveloperGuide.md:32]: oh didn't realise it made it bigger, just wanted to standardise it/ make the preview available in IntelliJ lol. I can revert the changes

File: docs/AboutUs.md (hunk 11 above)
Comment (bentanjunrong) [reply docs/AboutUs.md:34]: kk

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (bentanjunrong) [reply docs/DeveloperGuide.md:519]: how so?

File: docs/index.md (hunk 9 above)
Comment (bentanjunrong) [reply docs/index.md:6]: where do we get the badge from though? couldn't find it in our project files

**Comment by bentanjunrong** [comment]
Hi sirs @luminousleek @rohit0718, made most of the changes besides the unresolved comments/ and the `Model.java` diagram. Unfortunately I have multiple assignments due tonight so I'll have to ask that someone else pick up from wherever I left off 😢😢😢

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (bentanjunrong) [reply docs/DeveloperGuide.md:519]: Do you mean to follow the Use Case style numbering?

**Review by luminousleek — APPROVED** [review]
ok thanks sir i think we merge first and then @rohit0718 can touch it up?

**Review by rohit0718 — APPROVED** [review]
thanks! sure, ill settle the rest @bentanjunrong @luminousleek

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (rohit0718) [reply docs/DeveloperGuide.md:519]: nope haha just that some parts numberings are not incrementing

File: docs/index.md (hunk 9 above)
Comment (rohit0718) [reply docs/index.md:6]: u can get it from the github action workflow page!

**Pull request merged**